docs/sdks/client-sdks/flutter/intro.mdx

```diff
@@ -5,27 +5,27 @@ sidebar_position: 1
 
 import FeatureCard from "/src/components/FeatureCard";
 
-The Eppo SDK for Flutter Web allows you to manage feature flags and experiments in your Flutter Web applications.
+The Eppo SDK allows you to manage feature flags and experiments in your Dart and Flutter applications.
```

Comment:
note to self: 

- add information about minimal dart version (v3) 
- call out compatibility with web, android, io
- no restrictions on web
- minimum android sdk and flutter version
- minimum ios sdk and flutter version